### PR TITLE
Controlled Components / Populate Defaults from Store

### PR DESCRIPTION
**Pull request by rpurushotma**

### CODE REVIEW

File: gui/src/Users/components/UserAuthentication.tsx [hunk 1]
```diff
@@ -43,28 +43,28 @@ interface DispatchProps {
...
-  //realnameAttributeInput: HTMLInputElement;
-  groupDnInput: HTMLInputElement;
-  staticMemberAttributeInput: HTMLInputElement;
-  searchRequestSizeLimitInput: HTMLInputElement;
-  searchRequestTimeLimitInput: HTMLInputElement;
 
   constructor(params) {
     super(params);
     this.state = {
+      hostValue: "",
```
Comment (justinloyola) [inline gui/src/Users/components/UserAuthentication.tsx]: @vitreuz can you double check these defaults?

Comment (vitreuz) [inline gui/src/Users/components/UserAuthentication.tsx]: - portValue: "636"
- sslEnabled: True
- consistency on DC/dc
- staticMemberAttr: "memberUid"

**Comment by justinloyola** [comment]
LGTM.

**Comment by rpurushotma** [comment]
@justinloyola updated

**Pull request merged**